src/powershell/install-boxstarter.ps1

```diff
@@ -0,0 +1,43 @@
+# Install and configure Boxstarter
+# See: https://boxstarter.org/
+function Install-Boxstarter {
+    Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://boxstarter.org/bootstrapper.ps1'))
+    Get-Boxstarter -Force
+
+    # Fix verbosity issues with Boxstarter v3
+    # See: https://github.com/chocolatey/boxstarter/issues/501
+    $fileToFix = "${Env:ProgramData}\boxstarter\boxstarter.chocolatey\Chocolatey.ps1"
+    $offendingString = 'if ($val -is [string] -or $val -is [boolean]) {'
+    if ((Get-Content $fileToFix -raw) -contains $offendingString) {
+        $fixString = 'if ($val -is [string] -or $val -is [boolean] -or $val -is [system.management.automation.actionpreference]) {'
+        ((Get-Content $fileToFix -raw) -replace [regex]::escape($offendingString),$fixString) | Set-Content $fileToFix
+    }
+    $fileToFix = "${Env:ProgramData}\boxstarter\boxstarter.chocolatey\invoke-chocolatey.ps1"
+    $offendingString = 'Verbose           = $VerbosePreference'
+    if ((Get-Content $fileToFix -raw) -contains $offendingString) {
+        $fixString = 'Verbose           = ($global:VerbosePreference -eq "Continue")'
+        ((Get-Content $fileToFix -raw) -replace [regex]::escape($offendingString),$fixString) | Set-Content $fileToFix
+    }
+}
+
+function Set-BoxstarterConfiguration {
+    $Boxstarter.RebootOk = (-not $noReboots.IsPresent)
+    $Boxstarter.AutoLogin = $true
+    $Boxstarter.SuppressLogging = $True
+    $global:VerbosePreference = "SilentlyContinue"
+    Set-BoxstarterConfig -NugetSources "$desktopPath;.;https://www.myget.org/F/vm-packages/api/v2;https://chocolatey.org/api/v2"
```

Comment:
Please provide some informative commenting around the sources used here.

Reply:
@mcdonnnj resolved in [4b0ef07](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/4b0ef07a29b9de24ef2b7c94a7edab347181c92f)